packages/oml/oml.0.0.5/opam

```diff
@@ -10,8 +10,8 @@ install: [make "install"]
 remove: ["ocamlfind" "remove" "oml"]
 depends: [
   "ocamlfind" {build}
-  "lacaml"
+  "lacaml" {>= "7.2.3" & <= "7.6.2"}
```

Comment:
This is the incorrect constraint. `7.2.3` appears to build but `7.2.4` is missing a `.a`.